src/test/java/com/openpojo/reflection/impl/sampleclasses/AClassWithEquality.java

```diff
@@ -20,6 +20,9 @@
 /**
  * @author oshoukry
  */
+
+//TODO: Discuss why public class does not have hashCode implemented
```

Comment:
test classes are setup with just bare minimum required to exercise the feature under test.